README.md

```diff
@@ -5,6 +5,9 @@
   <a href="https://github.com/bytebeamio/rumqtt/actions/workflows/main.yml">
     <img alt="build status" src="https://github.com/bytebeamio/rumqtt/actions/workflows/main.yml/badge.svg">
```

Comment:
while we are at badges, and as you deleted `main.yml` workflow, how is this badge gonna work?

Reply:
This workflow runs for every commit to main. That is, when a pull request gets merged, this workflow will run.

Reply:
yup, but if i'm not wrong, the _badge_ checks the latest run status of workflow file with name `main.yml`.

and as there is no such workflow, the badge isn't going to work.

Reply:
Oh right. I'll fix that.

Reply:
As the workflow is run with every PR, if we do it like that, it indicates the status of PR.

so if I just open PR which fails to build, it is gonna show fail. this was the reason I did change it to `main.yml` previously, you can see the commit history :)

Reply:
I beg to differ.

```
[![CI](https://github.com/bytebeamio/rumqtt/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/bytebeamio/rumqtt/actions/workflows/ci.yml)
```

By default, the badge is going to show the status of the default branch, not for every pull request. If you want to be pedantic, you can include the name of the branch too, as I did in the above code block. I recommend you go through the [docs for github actions](https://docs.github.com/en/actions).